devops/scenario.py

```diff
@@ -246,62 +256,28 @@ def configure_agents(
     def configure_environment(
         self,
         vega: Union[VegaServiceNull, VegaServiceNetwork],
-        network: Network,
-        raise_datanode_errors: Optional[bool] = False,
-        raise_step_errors: Optional[bool] = False,
+        tag: Optional[str] = None,
+        random_state: Optional[np.random.RandomState] = None,
+        **kwargs,
     ) -> Union[MarketEnvironmentWithState, NetworkEnvironment]:
-        if network == Network.NULLCHAIN:
+        if kwargs["network"] == Network.NULLCHAIN:
```

Comment:
Let's do `kwargs.get` here and default it to non-nullchain

Reply:
Sounds good thanks, will update the default in the scenario runner as well to a non-nullchain network.